README.md

```diff
@@ -1,81 +1,160 @@
+
+## Why BPF in Kubernetes?
+
+Demand is increasing from both Kubernetes developers and users. Examples of BPF
+in Kubernetes include:
+
+- [Cilium](https://cilium.io/) and
+  [Calico](https://www.tigera.io/project-calico/) CNIs 
+- [Pixie](https://px.dev/): Open source observability
+- [KubeArmor](https://kubearmor.io/): Container-aware runtime security
+  enforcement system
+- [Blixt](https://github.com/Kong/blixt): Gateway API L4 conformance
+  implementation
+- [NetObserv](https://github.com/netobserv): Open source operator for network
+  observability 
+
+## Challenges for BPF in Kubernetes
+
+- Requires privileged pods.
+  - BPF-enabled apps require at least CAP_BPF permissions and potentially more
+    depending on the type of program that is being attached.
+  - Since the Linux capabilities are very broad it is challenging to constrain a
+    pod to the minimum set of privileges required. This can allow them to do
+    damage (either unintentionally or intentionally).
+- Handling multiple BPF programs on the same BPF hooks.
+  - Not all BPF hooks are designed to support multiple programs.
+  - Some software using BPF assumes exclusive use of a BPF hook and can
+    unintentionally eject existing programs when being attached. This can result
+    in silent failures and non-deterministic failures.
+- Debugging problems with deployments is hard.
+  - The cluster administrator may not be aware that BPF programs are being used
+    in a cluster.
+  - It is possible for some BPF programs to interfere with others in
+    unpredictable ways.
+  - SSH access or a privileged pod is necessary to determine the state of BPF
+    programs on each node in the cluster.
+- Lifecycle management of BPF programs.
+  - While there are libraries for the basic loading and unloading of BPF
+    programs, a lot of code is often needed around them for lifecycle
+    management.
+- Deployment on Kubernetes is not simple.
+  - It is an involved process that requires first writing a daemon that loads
+	  your BPF bytecode and deploying it using a DaemonSet.
+  - This requires careful design and intricate knowledge of the BPF program
+	  lifecycle to ensure your program stays loaded and that you can easily
+	  tolerate pod restarts and upgrades.
+  - In Bpf enabled K8s deployments today, the Bpf Program is often embedded into
+    the userspace binary that loads and interacts with it. This means there's no
+    easy way to have fine-grained versioning control of the bpfProgram in
+    relation to it's accompanying userspace counterpart.
+
+## What is bpfd?
+
+bpfd is a software stack that aims to make it easy to load, unload, modify and
+monitor BPF programs whether on a single host, or in a Kubernetes cluster. bpfd
+includes the following core components:
+
+  - bpfd: A system daemon that supports loading, unloading, modifying and
+	  monitoring of BPF programs exposed over a gRPC API.
+  - eBPF CRDS: That provide a way to express intent to load BPF programs
```

Comment:
You left the original line in and then added my suggestion as a new bullet. Delete this original bullet item.